lib/Chat/SystemMessage/Listener.php

```diff
@@ -220,6 +221,14 @@ protected function sendSystemGuestPermissionsMessage(RoomModifiedEvent $event):
 		}
 	}
 
+	protected function sendSystemMentionPermissionsMessage(RoomModifiedEvent $event): void {
```

Comment:
I think I wouldn't log this in the chat. We also don't log change of the default permissions.